app/views/catalog/access_panels/_in_collection.html.erb

```diff
@@ -11,23 +11,19 @@
             <%= parent_collection[:summary_display].join(', ') %>
           </div>
         <% end %>
-      </div>
-      <div class="panel-footer">
         <dl class="dl-invert">
           <% if parent_collection.collection_members.present? %>
-            <dt>Digital content</dt>
-            <dd><%= link_to_collection_members(pluralize(parent_collection.collection_members.total, 'item'), parent_collection) %></dd>
+            <dt>Digital collection</dt>
+            <dd><%= link_to_collection_members(pluralize(parent_collection.collection_members.total, 'digital item'), parent_collection) %></dd>
           <% end %>
-          <% if parent_collection.extent.present? %>
-            <dt><%= parent_collection.extent_label %></dt>
-            <dd><%= parent_collection.extent %></dd>
+          <% if parent_collection.extent_sans_format.present? %>
+            <dt>Physical collection</dt>
+            <dd><%= parent_collection.extent_sans_format %></dd>
```

Comment:
In doing some style tweaks w/ @jvine we came across this same data in a masthead.  While it's not the Record View per-se, it would be nice if we could update the content in the masthead to match this: https://github.com/sul-dlss/SearchWorks/blob/fc1e4c5b362789ea4cfdf62e18877a20b45c3fef/app/views/catalog/mastheads/_collection.html.erb#L16-L24

I was looking into doing that in my branch, but I would have to reproduce the `extent_sans_format` work, and that seemed like it might be more of a pain that to make the change here.

Reply:
👍 I've addressed this in #2200

![collection_masthead_after](https://user-images.githubusercontent.com/5402927/48806095-28feb780-ecce-11e8-8d27-9d51b2c2ca72.png)